compatibility/bazel_tools/testing.bzl

```diff
@@ -225,7 +225,9 @@ def sdk_platform_test(sdk_version, platform_version):
     sandbox = "@daml-sdk-{platform_version}//:daml".format(
         platform_version = platform_version,
     )
-    sandbox_args = ["sandbox"]
+    # We need to use weak seeding to avoid our tests timing out
+    # if the CI machine does not have enough entropy.
+    sandbox_args = ["sandbox", "--contract-id-seeding=testing-weak"]
```

Comment:
Don't you need a `--sandbox-option` here?

Reply:
No, `--sandbox-option` is for `daml start`. `daml sandbox` just calls the underlying JAR directly without passing any extra options.